cmd/sonarExecuteScan.go

```diff
@@ -195,6 +218,14 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 		log.Entry().WithError(err).Warning("no scan report found")
 		return nil
 	}
+
+	var serverUrl string
+
+	if len(config.ServerURL) > 0 {
+		serverUrl := config.ServerURL
+	} else {
+		serverUrl := taskReport.ServerURL
```

Comment:
I think = is sufficient here
```suggestion
		serverUrl = taskReport.ServerURL
```